packages/frontend/src/constants/index.ts

```diff
@@ -2,18 +2,22 @@ import { ChainId } from "@carrot-kpi/sdk";
 import { FunctionComponent, SVGProps } from "react";
 import { gnosis, sepolia } from "wagmi/chains";
 import { Chain } from "wagmi/chains";
-import { ReactComponent as EthereumLogo } from "../assets/chains/ethereum.svg";
-import { ReactComponent as GnosisLogo } from "../assets/chains/gnosis.svg";
+import EthereumLogo from "../icons/chains/ethereum";
+import GnosisLogo from "../icons/chains/gnosis";
 import { NavLink } from "react-router-dom";
 import { Link } from "react-router-dom";
 
 export const CARROT_KPI_FRONTEND_I18N_NAMESPACE = "@carrot-kpi/frontend";
 
 export interface AugmentedChain extends Chain {
-    logo: FunctionComponent<SVGProps<SVGSVGElement>>;
+    logo: FunctionComponent<
+        SVGProps<SVGSVGElement> & { title?: string | undefined }
+    >;
     iconBackgroundColor: string;
 }
 
+// FunctionComponent<SVGProps<SVGSVGElement> & { title?: string | undefined; }>
```

Comment:
We can remove these comments I guess.

Reply:
Addressed in 287eedfe63c6938294fe51b84384501ca05a2624